test_util/aws.py

```diff
@@ -87,7 +88,7 @@ def create(cls, stack_name, template_url, public_agents, private_agents,
             'PublicSlaveInstanceCount': str(public_agents),
             'SlaveInstanceCount': str(private_agents)}
         stack = boto_wrapper.create_stack(stack_name, template_url, parameters)
-        return cls(stack.stack.stack_id, boto_wrapper)
+        return cls(stack.stack.stack_name, boto_wrapper)
```

Comment:
Suggestion: Add a comment about that. That is definitely fun crazy boto behavior (although I can guess where the implementation bug is...)